pyproject.toml

```diff
@@ -2,29 +2,30 @@
 requires = ["setuptools", "wheel"]
 build-backend = "setuptools.build_meta"
 
-[project]
+[tool.poetry]
```

Comment:
All of these changes are because I kept geting this error:

`[tool.poetry] section not found in /Users/david/src/AgentStack/pyproject.toml`

It seems like maybe in more recent Python or Poetry versions (using Python 3.12 and Poetry 1.8.4), the `pyproject.toml` schema has changed?

Reply:
there's an issue installing the CLI with poetry, so I removed it here. we don't use poetry with agentstack, but the projects generated by the CLI currently do. I know this is confusing and it will be addressed in v0.2. you can revert this line for now!